### PR TITLE
VACMS-18701: updated composer dependencies

### LINKED ISSUE #18701: Doing mass composer.json update of packages to get dependapbot caught up.
## User Story or Problem Statement
Dependabot is way behind. This will catch us up in one PR and get us back to where we need to be. 

## Acceptance Criteria
- [ ] Most minor and patch-level updates are done.

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Close #18701 

## Testing done
Tested locally

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

Normally, for these updates, we depend on automated testing, so if all automated testing passes, they should be good. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [x] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/system.action.message_delete_action.yml [hunk 1]
```diff
@@ -1,13 +0,0 @@
-uuid: 83f77be8-80d0-42b9-9b6a-77492b18453f
```
Comment (timcosgrove) [inline config/sync/system.action.message_delete_action.yml:1]: Why is this deleted?

File: tests/cypress/integration/features/platform/permissions.feature [hunk 2]
```diff
@@ -13,8 +13,9 @@ Feature: Permissions
     And I should receive status code 200 when I request "/admin/config/search/redirect/migrate"
     And I should receive status code 200 when I request "/admin/config/search/redirect"
 
+  @ignore
   Scenario: Administer user role can add/edit users
-    Given I am logged in as a user with the "admnistrator_users" role
+    Given I am logged in as a user with the "administrator_users" role
```
Comment (timcosgrove) [inline tests/cypress/integration/features/platform/permissions.feature:18]: Just curious, how did our tests ever pass given this error? 🙂

Comment (edmund-dunn) [reply tests/cypress/integration/features/platform/permissions.feature:18]: Your guess is as good as mine. I talked this through with Jon. We are looking at a way to test these routes outside of cypress, to help get cypress more performant.

File: config/sync/system.action.message_delete_action.yml (hunk 1 above)
Comment (edmund-dunn) [reply config/sync/system.action.message_delete_action.yml:1]: Because the plugin was part of the eca module and was removed as part of the update. They had instructions to remove it. I verified we weren't using it.

**Review by timcosgrove — CHANGES_REQUESTED** [review]

**Review by timcosgrove — APPROVED** [review]
